.github/stale.yml

```diff
@@ -0,0 +1,18 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 30
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 10
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - security content
```

Comment:
`-` in yaml represents an array. To add a new entry, you'll want another list item here, otherwise this just translates into a string. this is matching on a "security content" label. Assuming these are different, you'd want this:

```yaml
exemptLabels:
  - security
  - content
```

Reply:
woot